alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from models import BaseModel
```

Comment:
Ensure that `BaseModel` is correctly defined in the `models` module and includes the SQLAlchemy models you want to manage with Alembic. This is necessary for the autogenerate feature to detect changes in the database schema.